src/domain/member.ts

```diff
@@ -50,7 +51,7 @@ export function updateMemberReputationWithValue(
 ): void {
   let member = getMember(address, daoAddress);
   member.reputation = value;
-  if (equals(member.reputation as BigInt, BigInt.fromI32(0))) {
+  if (equals(member.reputation, BigInt.fromI32(0)) && equals(member.tokens, BigInt.fromI32(0))) {
```

Comment:
does and why token holders are dao members ?

Reply:
Yes and because that's what Alchemy needed

Reply:
plus @jellegerbrandy for clarification here.

Reply:
No, I think it is clearer if we members collection only contanis the reputation holders. (So I'd prefer not to merge this change)